Template::Document->new({METADATA=>{'modtime'=>'1728930269','name'=>'/usr/local/cpanel/whostmgr/docroot/templates/store/product_purchase_failed.tmpl',},BLOCK=>sub {my$context=shift || die "template sub called without context\n";my$stash=$context->stash;my$output='';my$_tt_error;eval {BLOCK: {$output .= do {my$output='';$output .= '

';$stash->set('Whostmgr',$context->plugin('Whostmgr'));$output .= '

<link rel="stylesheet" type="text/css" href="';$output .= $stash->get(['Whostmgr',0,'find_file_url',['/templates/store/product_purchase.min.css' ]]);$output .= '" />

<div id="purchase-failed" class="callout callout-danger finished">
    <div class="callout-heading">
        ';$output .= $stash->get(['locale',0,'maketext',['Error' ]]);$output .= '
    </div>
    <div id="purchase-error">
        ';$output .= $stash->get(['data',0,'error',0]);$output .= '
    </div>
    <div id="purchase-error-detail" class="error-detail">
        ';$output .= $stash->get(['data',0,'error_detail',0,'html',0]);$output .= '
    </div>
    <div id="purchase-error-recommendation">
        ';$output .= $stash->get(['locale',0,'maketext',['If this problem persists, please [output,url,_1,open a Customer Service ticket,target,_blank], and reference the error above.','https://tickets.cpanel.net/submit/index.cgi?reqtype=cs' ]]);$output .= '
    </div>
    <div class="go-back">
        <a id="error-return-to-origin" href="';$output .= $stash->get(['data',0,'redirect_url',0,'html',0]);$output .= '">';$output .= $stash->get(['locale',0,'maketext',['Go back' ]]);$output .= '</a>
    </div>
</div>
';$context->include('master_templates/master.tmpl',{'theme'=>'bootstrap','header'=>$stash->get(['locale',0,'maketext',['[_1] purchase',$stash->get(['data',0,'product',0])]]),'content',$output })};$output .= '
'}};if ($@){$_tt_error=$context->catch($@,\$output);die$_tt_error unless$_tt_error->type eq 'return'}return$output},DEFBLOCKS=>{},});